### ISSUE TITLE
[6.0] When editing a story the app will always fully reload

### ISSUE THREAD

**Issue by dannyhw** [storybookjs/react-native#219]
**Describe the bug**

When editing a story fast reload is not working correctly.

This section of code was removed from the loadCsf file and might be related (line 251)

```
 if (m && m.hot && m.hot.dispose) {
      ({ previousExports = new Map() } = m.hot.data || {});
      m.hot.dispose((data) => {
        loaded = false;
        // eslint-disable-next-line no-param-reassign
        data.previousExports = previousExports;
      });
      m.hot.accept();
    }
```

The dispose part is not need however perhaps maintaining the hot.accept part would be useful... this needs some investigation probably.
```
 if (m && m.hot) {
      m.hot.accept();
  }
 
```

**To Reproduce**

1. run example app
2. edit a story file and save it or just save with no changes
2. app fully reload

**Expected behavior**

Fast reload should be used such that only what changed gets reloaded.

**Comment by lauriharpf** [comment]
I could take a stab at this! :)

**Comment by dannyhw** [comment]
I've moved this to alpha2 milestone. I think that we can go ahead without fast reload for now considering that the full reload now takes you back to the previous story. This seems like a difficult issue to solve so I'd rather not delay on the first alpha due to this.

**Comment by dannyhw** [comment]
fixed in #384
